### ISSUE TITLE
Terms Cloud elements overflow page

### ISSUE THREAD

**Issue by sap218** [kaiiiz/hugo-theme-monochrome#57]
Hi! I am using Monochrome for my site as a submodule: https://github.com/sap218/hugo-test/

and when I pulled in monochrome recent version, I noticed my terms cloud seems to be broken and overflow the page (see screenshot / https://sap218.uk/categories/)

***

![overflow](https://github.com/user-attachments/assets/46e277d0-9c45-454f-8ca7-4901297cadc4) 

***

I think it might be [related to line 4](https://github.com/kaiiiz/hugo-theme-monochrome/compare/960a29aca460b100a9191950280c0e71f18323d8...ee549d5018a8fe105031ff7b9263824a3f0be54c#diff-061e48e296a7bf8aaf372e33e7ffe4182cde8492ba94f1c0f3cb3afd8a3f5ff0) in: assets/scss/components/terms-cloud.scss

```
- line-height: 1.7rem;
+ text-wrap: nowrap;
```

when I changed this locally, it seems to fix the issue

***

![local](https://github.com/user-attachments/assets/cd3f57ef-4316-4789-a351-236b4aa2558e)

***

Let me know if there is an easy fix I can do on my end or if you need any more information from me! 

Thank you for the theme! 😄

**Comment by kaiiiz** [comment]
Your solution should be sufficient! Thanks for reporting the bug.

I'm quite uncomfortable for the line breaking of the counter (e.g. `Ontology` and `(8)` in the above example), but it is far better than the nowrap ;).

The fix has been applied in v4.0.0.

**Comment by Nightfirecat** [comment]
Alternatively, this could probably be fixed by applying the following to the `<div>` which wraps the `.term-item` elements:

```css
div { /* needs proper class or selector */
	align-items: end;
	display: flex;
	flex-wrap: wrap;
}
```

That would prevent the links from breaking between two lines, but still allow them to wrap to however many lines are needed to fit them all.
